kernel/data-pipeline/api/src/main/java/org/apache/shardingsphere/data/pipeline/api/check/consistency/DataConsistencyCalculateParameter.java

```diff
@@ -73,4 +73,9 @@ public final class DataConsistencyCalculateParameter {
      * Previous calculated result will be transferred to next call.
      */
     private volatile Object previousCalculatedResult;
+    
+    /**
+     * The position value of data check.
+     */
+    private final Object dataCheckPositionValue;
```

Comment:
`dataCheckPositionValue` could be `tableCheckPosition`, keep it consistency with other places